src/sass-builder.js

```diff
@@ -46,26 +49,31 @@ sass.importer((request, done) => {
 });
 
 export default (loads, compileOpts) => {
-  return new Promise((resolve, reject) => {
-    loads.forEach((load) => {
-      // TODO Support different load addresses
+  const stubDefines = loads.map(load => {
+    return `${(compileOpts.systemGlobal || 'System')}\.register('${load.name}', [], false, function() {});`;
+  }).join('\n');
+
+  const compile_promise = load => {
+    return new Promise((resolve, reject) => {
       urlBase = load.address;
+      const options = {
+        style: sass.style.compressed,
+        indentedSyntax: urlBase.split('.').slice(-1)[0] == 'sass'
```

Comment:
You can use the shiny new [String.prototype.endsWith()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/endsWith) method here.